src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -125,17 +153,20 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
                 ] = data_values.MilitaryEmployer.EMPLOYER_ID
                 pop = self._update_employer_metadata(pop, military_index)
```

Comment:
You should move the code common to both branches of the if statement outside and call them once after the `if`. This includes setting the personal income propensity and calling `population_view.update`. You could also move the employer propensity outside, though that would mean drawing for the newborns rather than just setting 0.0.

Reply:
I am not sure I understand this here.  The if branch is based on simulation initialization vs on time step when newborns are initialized.  Wouldn't moving any assignment for newborns to the if branch never be hit because that happens after start time?